torch/quantization/fx/quantization_patterns.py

```diff
@@ -378,6 +378,27 @@ def convert(self, quantizer, node, load_arg, debug=False, convert_custom_config_
             load_arg(quantized=[0])(self.bn_node.args),
             load_arg(quantized=False)(self.bn_node.kwargs))
 
+@register_quant_pattern(torch.nn.Embedding)
+class Embedding(QuantizeHandler):
+    def __init__(self, quantizer, node):
+        super().__init__(quantizer, node)
+        assert node.op == 'call_module'
+        self.emb_node = node
+        self.emb = quantizer.modules[self.emb_node.target]
+
+    def convert(self, quantizer, node, load_arg, debug=False, convert_custom_config_dict=None):
+        qconfig = quantizer.qconfig_map[node.name]
+        assert not activation_is_statically_quantized(qconfig)
+        qemb = nnq.Embedding
```

Comment:
isn't this dynamic quant? why is this nnq instead of nnqd?

Reply:
It is weight only quantization. Main use case for embedding quantization is recommender models which use static quant to quantize embeddings and FC (mainly). So we decided to keep this in nnq for ease of use.

Reply:
I see, now that we changed api in fx, I feel we should probably put this in a separate namespace to make it clear that it is weight only quantization
e.g. `nn.quantized.weight_only`